### PR TITLE
[BACKEND ONLY] save user confirmation for permanent embedding of iframe

### PR DESCRIPTION
**Pull request by ogerly**
## 🍰 Pullrequest
save user confirmation for permanent embedding of iframe

## allowEmbedIframes 

created in the database

### CODE REVIEW

**Comment by appinteractive** [comment]
What if I want to decide based on provider?

**Comment by ogerly** [comment]
> What if I want to decide based on provider?

I've been thinking about that, too. 
basically we have a list that is manageable. 
if it should be nice. must be with the confirmations a list to select.

**Comment by roschaefer** [comment]
@appinteractive @ogerly [KISS](https://en.wikipedia.org/wiki/KISS_principle) please! One setting for all or no provider is enough, in my opinion.

Every piece of functionality or programming logic is adding more bugs: If we have a list, we must make sure that it works as expected. If we don't have a list, then it's a lot easier.

**Comment by appinteractive** [comment]
I agree with @roschaefer, my comment was related with saving the setting as it was not clear to me if it gets forcebly saved when I show it once.

File: backend/src/models/User.js [hunk 1]
```diff
@@ -104,4 +104,8 @@ module.exports = {
     target: 'Location',
     direction: 'out',
   },
+  allowEmbedIframes: {
+    type: 'boolean',
```
Comment (roschaefer) [inline backend/src/models/User.js]: You could even think about using a `date` here. The `false` would be if it's null.

Comment (mattwr18) [reply backend/src/models/User.js]: I like that suggestion... maybe it's useful to know at some point when the user opted in to this "feature"

Comment (mattwr18) [reply backend/src/models/User.js]: Although, it would have to be the `String` representation of a `datetime`, no @roschaefer?

**Comment by ogerly** [comment]
> @ogerly looks good but I would prefer to have a user-facing feature before we merge changes to the code. It would be confusing to see these changes if they have no effect right now.

 

this PR and the PR https://github.com/Human-Connection/Human-Connection/pull/1682 I would like to have together already in it around the issue to convert https://github.com/Human-Connection/Human-Connection/issues/1684

**Comment by ogerly** [comment]
the PR is integrated here https://github.com/Human-Connection/Human-Connection/pull/1685

**Pull request closed without merging**